src/WebJobs.Script.WebHost/Management/WebFunctionsManager.cs

```diff
@@ -33,9 +31,10 @@ public class WebFunctionsManager : IWebFunctionsManager
         private readonly IFunctionMetadataManager _functionMetadataManager;
         private readonly IHostFunctionMetadataProvider _hostFunctionMetadataProvider;
         private readonly IOptionsMonitor<LanguageWorkerOptions> _languageWorkerOptions;
+        private readonly IOptions<FunctionsHostingConfigOptions> _hostingConfigOptions;
```

Comment:
Reviewing on mobile, but can you please check the scope of this service (web/jobhost) and consider using IOptionsMonitor here instead? Hosting config will be dynamically updated and the options refreshed when changed, but using IOptions will prevent you from getting updated states

Reply:
Switched to `IOptionsMonitor<FunctionsHostingConfigOptions`.

We have 16 other instances using `IOptions<FunctionsHostingConfigOptions>`
https://github.com/search?q=repo%3AAzure%2Fazure-functions-host+IOptions%3CFunctionsHostingConfigOptions%3E&type=code&p=2